src/components/settings/forms/SectorOld.vue

```diff
@@ -0,0 +1,510 @@
+    },
+
+    areAllFieldsFilled() {
+      const { name, managers, workingDay, maxSimultaneousChatsByAgent } =
+        this.sector;
+      this.hourValidate(workingDay);
+      return !!(
+        name.trim() &&
+        managers.length > 0 &&
+        workingDay?.start &&
+        workingDay?.end &&
+        this.validHour &&
+        maxSimultaneousChatsByAgent
+      );
+    },
+
+    hourValidate(hour) {
+      const inicialHour = hour.start;
+      const finalHour = hour.end;
+      if (inicialHour > finalHour) {
+        this.validHour = false;
+        this.message = 'Horário de início não pode ser maior que horário final';
+      } else {
+        this.validHour = true;
+      }
+    },
+
+    async deleteSector(sectorUuid) {
+      try {
+        await this.actionDeleteSector(sectorUuid);
+        this.openModalDelete = false;
+        this.$router.push({ name: 'sectors' });
+        unnnic.unnnicCallAlert({
+          props: {
+            text: this.$t('sector_deleted_success'),
+            type: 'success',
+          },
+          seconds: 5,
+        });
+      } catch (error) {
+        console.log(error);
+        this.openModalDelete = false;
+        unnnic.unnnicCallAlert({
+          props: {
+            text: this.$t('sector_delete_error'),
+            type: 'error',
+          },
+          seconds: 5,
+        });
+      }
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.form-sector {
+  .form-section {
+    & + .form-section {
+      margin-top: 1.5rem;
+    }
+
+    & .inline-input-and-button {
+      & *:first-child {
+        flex: 1 1;
+      }
+
+      display: flex;
+      gap: $unnnic-spacing-stack-sm;
+      align-items: flex-end;
+    }
+
+    .title {
+      &--lg {
+        font-size: $unnnic-font-size-title-sm;
+      }
+
+      font-weight: $unnnic-font-weight-bold;
+      color: $unnnic-color-neutral-dark;
+      font-size: $unnnic-font-size-body-lg;
+      line-height: 1.5rem;
+
+      margin-bottom: 1rem;
+    }
+
+    &__switch__container {
+      display: flex;
+      align-items: center;
+
+      .unnnic-tooltip {
+        display: flex;
+      }
+    }
+
+    &__inputs {
+      display: grid;
+      gap: $unnnic-spacing-stack-sm;
+      grid-template-columns: 1fr 1fr;
+
+      &--fill-w {
+        grid-column: span 2;
+      }
+    }
+
+    .unnnic-switch {
+      align-items: center;
+    }
+
+    &__handlers {
+      margin-top: $unnnic-spacing-md;
+      button {
+        width: 100%;
+      }
+    }
+  }
+
+  .input-time {
+    background: #fff;
+    border: 0.0625rem solid #e2e6ed;
+    border-radius: 0.25rem;
+    color: #4e5666;
+    font-weight: 400;
+    font-family: Lato;
+    box-sizing: border-box;
+    width: 100%;
+    font-size: 0.875rem;
+    line-height: 1.375rem;
+    padding: 0.75rem 1rem;
+    cursor: text;
+  }
+  .label-working-day {
+    font-weight: 400;
+    line-height: 1.375rem;
+    font-size: 0.875rem;
+    color: #67738b;
+    margin: 0.5rem 0;
```

Comment:
You can use the token variables we have in unnnic. Here are all the tokens
https://www.figma.com/design/YbvOpVPUBxTSEzg4J8pXKu/Weni---unnnic?node-id=4674-21107&m=dev